src/Components/Modal/ModalWrapper.tsx

```diff
@@ -96,8 +97,12 @@ export class ModalWrapper extends React.Component<
             unmountOnExit
             timeout={{ enter: 10, exit: 200 }}
           >
-            <ModalContainer isWide={isWide} image={image}>
-              <ModalInner>{children}</ModalInner>
+            <ModalContainer
+              isResponsive={isResponsive}
+              isWide={isWide}
+              image={image}
+            >
+              <ModalInner isResponsive={isResponsive}>{children}</ModalInner>
```

Comment:
I think we have other components built off of `Modal` / `ModalWrapper` other than `ErrorModal` and was concerned that they would no longer look as they should, as the responsive styling is pretty different than than the current styling, i.e. it's no longer full height and  width. We could also do this the other way around maybe - make the responsive styling the new default, and update the other modals to pass something like a `nonResponsive` prop.

Reply:
Maybe this is another naming thing 😅since technically they're both responsive, just different designs. Should we use something like `fullscreenResponsiveModal={true}` for the existing one and `false` for the error one?